.travis.yml

```diff
@@ -15,4 +15,5 @@ install:
     - "pipenv sync -d"
 
 script:
+    - "pipenv run autoformat"
```

Comment:
Why should Travis-CI run autoformat, he can't push those changes to GitHub anyway, that's the point of Travis-CI, it should only check things, not actually change them.  This should be removed.
```suggestion
```